tests/metrics/network/network-metrics-iperf3.sh

```diff
@@ -58,24 +62,17 @@ function iperf3_jitter {
 	rm -f $jitter_result
 }
 
-function iperf3_bidirectional_bandwidth_remote_local {
-	setup
-	bidirectional_bandwidth_result=$(mktemp)
-	$DOCKER_EXE run -d --name=iperf3-server ${image} bash -c "mount -t ramfs -o size=20M ramfs /tmp && iperf3 -p ${port} -s" > /dev/null && \
-	server_address=`$DOCKER_EXE inspect --format "{{.NetworkSettings.IPAddress}}" $($DOCKER_EXE ps -ql)` && \
-	$DOCKER_EXE run -ti --rm --name=iperf3-client ${image} bash -c "mount -t ramfs -o size=20M ramfs /tmp && iperf3 -c ${server_address} -d -t ${time}" > "$bidirectional_bandwidth_result"
-	total_bidirectional_bandwidth=`cat $bidirectional_bandwidth_result | tail -n 3 | head -1 |  awk '{print $(NF-2), $(NF-1)}'`
-	$DOCKER_EXE rm -f iperf3-server > /dev/null
-	rm -f $bidirectional_bandwidth_result
-}
-
-function iperf3_bidirectional_bandwidth_local_remote {
+# Run bi-directional TCP test, and extract results for both directions
+function iperf3_bidirectional_bandwidth_client_server {
 	setup
 	bidirectional_bandwidth_result=$(mktemp)
 	$DOCKER_EXE run -d --name=iperf3-server ${image} bash -c "mount -t ramfs -o size=20M ramfs /tmp && iperf3 -p ${port} -s" > /dev/null && \
 	server_address=`$DOCKER_EXE inspect --format "{{.NetworkSettings.IPAddress}}" $($DOCKER_EXE ps -ql)` && \
 	$DOCKER_EXE run -ti --rm --name=iperf3-client ${image} bash -c "mount -t ramfs -o size=20M ramfs /tmp && iperf3 -c ${server_address} -d -t ${time}" > "$bidirectional_bandwidth_result"
-	total_bidirectional_bandwidth=`cat $bidirectional_bandwidth_result | tail -n 4 | head -1 | awk '{print $(NF-3), $(NF-2)}'`
+echo "Data:"
+cat $bidirectional_bandwidth_result
```

Comment:
These 2 lines need an indent.

General comment: these lines are very long - we should at some point try to fold them to fit within the constrains of a smaller terminal (also makes the code easier to maintain).